src/ServiceControl/Monitoring/Web/ApiModule.cs

```diff
@@ -13,13 +15,16 @@ public class EndpointUpdateModel
     public class ApiModule : BaseModule
     {
         public EndpointInstanceMonitoring Monitoring { get; set; }
+        public GetKnownEndpointsApi KnownEndpointsApi { get; set; }
 
         public ApiModule()
         {
             Get["/heartbeats/stats"] = _ => Negotiate.WithModel(Monitoring.GetStats());
 
             Get["/endpoints"] = _ => Negotiate.WithModel(Monitoring.GetEndpoints());
 
+            Get["/endpoints/known", true] = (_, token) => KnownEndpointsApi.Execute(this, NoInput.Instance);
```

Comment:
There's a `NoInput` extension method so you shouldn't have to specify this right?

Reply:
Addressed in https://github.com/Particular/ServiceControl/pull/1101/commits/2c9af283672d284d33387a70048bad6acbac570e